app/constitution/actions.js

```diff
@@ -797,22 +797,6 @@ const actions = new Map([
         checkType(args.issuer, "string", "issuer");
         checkType(args.auto_refresh, "boolean?", "auto_refresh");
         checkType(args.ca_cert_bundle_name, "string?", "ca_cert_bundle_name");
-        checkEnum(args.key_filter, ["all", "sgx"], "key_filter");
```

Comment:
You should also remove other SGX-related actions from the constitution, like `add_node_code`, `remove_node_code`, etc.

Reply:
removed `add_node_code`, `remove_node_code` please comment if anything else needs to be removed